BlazarData/src/main/resources/schema.sql

```diff
@@ -188,3 +188,7 @@ CREATE TABLE branch_settings (
   "interProjectBuildOptIn" TINYINT(1),
   PRIMARY KEY ("branchId")
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
Any MySQL-specific settings need to done using an `ALTER` after-the-fact in a MySQL-specific changeset, [for example](https://github.com/HubSpot/Blazar/blob/37a138384f56928fc9653f1cf678336b60561169/BlazarData/src/main/resources/schema.sql#L96-L97)